### ISSUE TITLE
Severe misalignment between image and mask after random distortion

### ISSUE THREAD

**Issue by VincentYCYao** [mdbloice/Augmentor#188]
related issues: #173 #184
**This is a bug**
```python
def augmentation(x_root, y_root):
    p = Augmentor.Pipeline(x_root)
    p.ground_truth(y_root)

    # TODO: Bug fix: Augmentor can not synchronize the random distortion between image and label
    p.random_distortion(probability=1, grid_width=4, grid_height=4, magnitude=8)

    # save augmented images
    num_of_samples = int(1e2)
    p.sample(num_of_samples)
```
![distored_mask](https://user-images.githubusercontent.com/49467070/60991715-a8d0b380-a37d-11e9-8942-32cb6a4220c0.jpg)
![distore_image](https://user-images.githubusercontent.com/49467070/60991735-b128ee80-a37d-11e9-947a-474e2f9959e7.jpg)

**Comment by mdbloice** [comment]
Hi @VincentYCYao, sorry for the delay checking this, I have not been able to dedicate much time for Augmentor over the past few days. I will check this immediately, and try to see what might be happening.

**Comment by mdbloice** [comment]
Hi @VincentYCYao, this has now been resolved as of version 0.2.6. Just upgrade via pip (`pip install Augmentor -U`) 

Thanks for pointing out the issue!

**Comment by VincentYCYao** [comment]
@mdbloice Thanks a lot!
